packages/paste-style-props/src/constants.ts

```diff
@@ -1,6 +1,6 @@
 import RawJSON from '@twilio-paste/design-tokens/dist/tokens.raw.json';
 
-const camelCase = require('lodash/camelcase');
+const camelCase = require('lodash/camelCase');
```

Comment:
suggestion: pretty sure this can be `import camelCase from 'lodash/camelCase'` (which is the preferred way to import stuff)

Reply:
I don't exactly know _why_, but there are some spots where it won't build if I convert it to an `import` statement. For this latest set of changes, I converted all of the `require` lines to `import`, then systematically changed them back to `require` if they made the build fail. Any of them that have `require` right now have been checked and are like that (just checked this one again to be extra sure). I haven't figured out why and am very curious about it.